Widgets/RoomView.cpp

```diff
@@ -85,18 +85,16 @@ void RoomView::paintTiles(QPainter& painter) {
     bkg = bkg->GetSubModel<MessageModel*>(TreeNode::kBackgroundFieldNumber);
     if (!bkg) continue;
 
+    MessageModel* tile = _sortedTiles->data(_sortedTiles->index(row, 0)).value<MessageModel*>();
```

Comment:
Should we log/check this? Also, should we check if the QVariant is valid before taking the value?